tests/upgrades/test_usergroup.py

```diff
@@ -26,72 +26,74 @@
 from robottelo.constants import LDAP_SERVER_TYPE
 
 
-@pre_upgrade
-def test_pre_create_usergroup_with_ldap_user(request):
-    """Create Usergroup in preupgrade version.
-
-    :steps:
-        1. Create ldap auth pre upgrade.
-        2. Login with ldap User in satellite and logout.
-        3. Create usergroup and assign ldap user to it.
-
-    :expectedresults: The usergroup, with ldap user as member, should be created successfully.
+class TestUserGroupMembership:
     """
-    authsource = entities.AuthSourceLDAP(
-        onthefly_register=True,
-        account=settings.ldap.username,
-        account_password=settings.ldap.password,
-        base_dn=settings.ldap.basedn,
-        groups_base=settings.ldap.grpbasedn,
-        attr_firstname=LDAP_ATTR['firstname'],
-        attr_lastname=LDAP_ATTR['surname'],
-        attr_login=LDAP_ATTR['login_ad'],
-        server_type=LDAP_SERVER_TYPE['API']['ad'],
-        attr_mail=LDAP_ATTR['mail'],
-        name=request.node.name + "_server",
-        host=settings.ldap.hostname,
-        tls=False,
-        port='389',
-    ).create()
-    assert authsource.name == request.node.name + "_server"
-    sc = ServerConfig(
-        auth=(settings.ldap.username, settings.ldap.password),
-        url=f'https://{settings.server.hostname}',
-        verify=False,
-    )
-
-    with pytest.raises(HTTPError):
-        entities.User(sc).search()
-    user_group = entities.UserGroup(name=request.node.name + "_user_group").create()
-    user = entities.User().search(query={'search': f'login={settings.ldap.username}'})[0]
-    user_group.user = [user]
-    user_group = user_group.update(['user'])
-    assert user.login == user_group.user[0].read().login
-
-
-@post_upgrade(depend_on=test_pre_create_usergroup_with_ldap_user)
-def test_post_verify_usergroup_membership(request):
-    """After upgrade, check the LDAP user's(that we created before the upgrade) existence and
-    their functionality.
-
-    :steps:
-        1. verify ldap user(created before upgrade) is part of user group.
-        2. Update ldap auth.
-
-    :expectedresults: After upgrade, user group membership should remain the same and LDAP
-    auth update should work.
+    Usergroup membership should not lost after upgrade.
```

Comment:
```suggestion
    Usergroup membership should exists after upgrade.
```

Reply:
s/exists/exist/
How about (for emphasis):
Usergroup membership should still exist after upgrade.

Reply:
Fixed.